src/sentry/snuba/discover.py

```diff
@@ -55,7 +55,7 @@ def is_real_column(col):
     Return true if col corresponds to an actual column to be fetched
     (not an aggregate function or field alias)
     """
-    if col in FIELD_ALIASES:
+    if col in FIELD_ALIASES or col.strip("()") in FIELD_ALIASES:
```

Comment:
Hmm would it be better to use `match.group("function") in FIELD_ALIASES` instead? I think strip could be dangerous since it does stuff like:
```
"(test(".strip("()") == "test"
```

Reply:
I don't think there's a case where someone could break something by doing `(p95(()))` or whatever with this, since the column will get filtered if it matches so we won't send bad columns on either.